doc/content/getting-started/programming-software/st-examples/openocd/_index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "OpenOCD"
+description: ""
+weight: 3
+distributions: null
+---
+
+[OpenOCD](http://openocd.org/) is commonly used for on-chip debugging, but can also be used to easily program executables via a CLI.
+
+<!--more-->
+
+## Steps to reproduce
+
+To program a binary executable, run the following command:
```

Comment:
```suggestion
To program a binary executable, run the following command:

```

Reply:
Where do you run this command?

Reply:
In the terminal, just like make

Reply:
If you're talking about directory location, then it doesn't matter as one of the inputs it the binary file path

Reply:
ok, then just say "run the following command in your terminal" and that's it